src/lib.rs

```diff
@@ -421,23 +421,39 @@ where
     }
 }
 
-impl<T: PrimInt + Integer> std::fmt::Display for GaussianInt<T> {
+impl<T: PrimInt + Integer + Signed + std::fmt::Display> std::fmt::Display for GaussianInt<T> {
```

Comment:
The only problem is here. We really should be able to print `GaussianInt<usize>`.